crates/foundry-ui/src/components/mod.rs

```diff
@@ -0,0 +1,8 @@
+//! This module provides various ready to use message types for use with
+//! a [`Ui`][crate::Ui].
+
+pub use tagged::*;
+pub use typed::*;
+
+mod tagged;
+mod typed;
```

Comment:
If we are reexporting whole modules why not just do this?
```suggestion
pub mod tagged;
pub mod typed;
```